.github/workflows/release.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         env:
           # Using self-hosted runners so use local cache for sccache and
           # not SCCACHE_GHA_ENABLED.
-          RUSTC_WRAPPER: "sccache"
+          # RUSTC_WRAPPER: "sccache"
```

Comment:
why remove?

Reply:
Cause the net benefit in this case is 0 and it would take me more time to make it work for windows.

Windows is not set up the same way so it starts complaining. Instead of doing a custom unset for windows or separate job, simply disabling it works as well. 
These are jobs that are only rarely run manually and on tagged releases. They run with a different build profile so those caches we have from PRs are worth nothing in this case, might even be net negative where it pushes out some of those deps. By the time we do a follow up release, the deps are incredibly stale and most likely pushed out due to storage limits.